acl_upgrade.php

```diff
@@ -839,6 +840,45 @@
     // (will also place in the appropriate group and CREATE a new group if needed)
     echo "<BR/><B>Adding ACLs(Access Control Lists) and groups</B><BR/>";
 
+    //Alter ACLs here (will return the ACL ID of newly created or already existant ACL)
+    echo "<BR/><B> Altering ACLs(Access Control Lists) and groups</B><BR/>";
+    AclExtended::shiftAcl($accounting_view, 'Accounting', 'patients', 'Patients', 'pat_rep', 'Patient Report', 'view');
+    AclExtended::shiftAcl($frontoffice_view, 'Front Office', 'patients', 'Patients', 'pat_rep', 'Patient Report', 'view');
+    AclExtended::shiftAcl($frontoffice_wsome, 'Front Office', 'patients', 'Patients', 'trans', 'Transactions (write,wsome optional)', 'wsome');
+    AclExtended::shiftAcl($frontoffice_wsome, 'Front Office', 'patients', 'Patients', 'notes', 'Patient Notes (write,wsome optional)', 'wsome');
```

Comment:
would put these in below category `Update the ACLs`

Otherwise:
![2DV-2](https://user-images.githubusercontent.com/278968/153703444-c13ccd97-6e07-48d9-b5a7-fa23a7c062b4.gif)